src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -79,6 +80,27 @@ public static EntityPlayer getPlayerFromUUID(@NotNull World world, @NotNull UUID
         return players;
     }
 
+    //todo: document
+    @NotNull
+    public static List<EntityPlayer> getPlayersFromPermPlayer(@NotNull List<Permissions.Player> players)
+    {
+        @NotNull final List<EntityPlayer> playerList = new ArrayList<>();
+
+        for (@NotNull Permissions.Player player : players)
+        {
+            playerList.add(ServerUtils.getPlayerFromPermPlayer(player));
+        }
+
+        return playerList;
+    }
+
+    //todo: document!
+    @Nullable
+    public static EntityPlayer getPlayerFromPermPlayer(@NotNull Permissions.Player player)

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)